src/Theme/AdminNegotiator.php

```diff
@@ -46,7 +46,7 @@ class AdminNegotiator extends AdminNegotiatorBase {
    * @param \Drupal\og\OgAccessInterface $og_access
    *   The og access service.
    */
-  public function __construct(AccountInterface $user, ConfigFactoryInterface $config_factory, EntityManagerInterface $entity_manager, AdminContext $admin_context, SiteManagerInterface $site_manager, OgAccessInterface $og_access) {
+  public function __construct(AccountInterface $user, ConfigFactoryInterface $config_factory, EntityTypeManagerInterface $entity_type_manager, AdminContext $admin_context, SiteManagerInterface $site_manager, OgAccessInterface $og_access) {
     parent::__construct($user, $config_factory, $entity_manager, $admin_context);
```

Comment:
Add missing $entity_type_manager:
```suggestion
    parent::__construct($user, $config_factory, $entity_type_manager, $admin_context);
```
@sandipt27, note that github allows you to make inline suggestions (See number 6 on https://help.github.com/en/articles/commenting-on-a-pull-request).

Also have a look at https://help.github.com/en/articles/creating-a-pull-request-from-a-fork

So if you like to contribute, fork this project and create pull requests to suggest changes.

Good luck!